package.json

```diff
@@ -51,14 +51,15 @@
     "lint:css": "stylelint './src/**/*.{ts,tsx}'",
     "prettier": "prettier --write src/**/*.{ts,tsx} test/**/*.{ts,tsx,js} manifest/**/*.js webpack/*.js release.*.js",
     "verify": "yarn test && yarn typecheck && yarn lint && yarn build:chromium:production && yarn build-storybook",
+    "verify:quick": "yarn test && yarn typecheck && yarn lint",
```

Comment:
does the front build take a long time? it would be safer to keep it, no?

Reply:
`yarn test` 31s
`yarn typecheck` 25s
`yarn lint` 22s
`yarn build:chromium:production`  1m18s
`yarn build-storybook` 1m36s

Reply:
The CI checks that anyway …